runelite-client/src/main/java/net/runelite/client/plugins/devtools/DevToolsOverlay.java

```diff
@@ -233,10 +237,25 @@ private void renderTileObjects(Graphics2D graphics)
 				{
 					renderDecorObject(graphics, tile, player);
 				}
+
+				if (plugin.isToggleTileLocation())
+				{
+					renderTileTooltip(graphics, tile);
```

Comment:
There is a client api method already to return which tile the mouse is over, the tile highlight plugin uses it.